docs/src/pages/examples/memory/memory-todo-agent.mdx

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
Can we rename the slug and example to `/short-term-working-memory`? That is the higher-order bit, not the todo agent paty.

Reply:
*part